app/models/assigned_df_scope_limitation.rb

```diff
@@ -0,0 +1,2 @@
+class AssignedDfScopeLimitation < ApplicationProceedingTypesScopeLimitation
```

Comment:
Is this missing a belongs to? Because wouldn't it need to be deleted if the parent is deleted?